docs/setting-up-a-validator.md

```diff
@@ -175,12 +207,53 @@ journalctl -u orchestrator.service -f --output cat
 journalctl -u geth.service -f --output cat
 ```
 
-## Wait for it
+## Observe Sync Status and Time Remaining
+You will need to wait for your Gravity Bridge node and Ethereum Node to fully sync before progressing in the instructions </br>
+### Ethereum
+You can view the status of your Ethereum node by issuing the following command: </br> 
+```
+curl -H "Content-Type:application/json" -X POST -d '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' http://127.0.0.1:8545
+```
+When result is 'false' that means it is now synced ('true' is not synced).
```

Comment:
this is great content, I think we need some spaces to for markdown lint to pass though.

Reply:
linter now passed